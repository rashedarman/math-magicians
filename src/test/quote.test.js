import { render } from '@testing-library/react';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import QuotePage from '../pages/Quote';

it('Renders quote page', () => {
  const result = render(
    <BrowserRouter>
      <QuotePage />
    </BrowserRouter>,
  );
  expect(result).toMatchSnapshot();
});
