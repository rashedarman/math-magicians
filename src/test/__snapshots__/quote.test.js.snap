// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders quote page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        id="main-layout"
      >
        <header>
          <div
            class="container"
          >
            <nav>
              <h1>
                Math Magicians
              </h1>
              <ul>
                <li>
                  <a
                    href="/"
                  >
                    Home
                  </a>
                </li>
                <li>
                  <a
                    href="/calculator"
                  >
                    Calculator
                  </a>
                </li>
                <li>
                  <a
                    href="/quote"
                  >
                    Quote
                  </a>
                </li>
              </ul>
            </nav>
          </div>
        </header>
        <div
          class="children container"
        >
          <p
            style="font-size: 2rem; font-weight: 500; text-align: center;"
          >
            Mathematics is not about numbers, equations, computations, or alogrithms; it is about understanding. -William Paul Thuston
          </p>
        </div>
        <footer
          class="copyrightText"
        >
          <p
            class="copyright-text"
          >
            Copyright © Math Magicians 2023. All rights reserved.
          </p>
        </footer>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      id="main-layout"
    >
      <header>
        <div
          class="container"
        >
          <nav>
            <h1>
              Math Magicians
            </h1>
            <ul>
              <li>
                <a
                  href="/"
                >
                  Home
                </a>
              </li>
              <li>
                <a
                  href="/calculator"
                >
                  Calculator
                </a>
              </li>
              <li>
                <a
                  href="/quote"
                >
                  Quote
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </header>
      <div
        class="children container"
      >
        <p
          style="font-size: 2rem; font-weight: 500; text-align: center;"
        >
          Mathematics is not about numbers, equations, computations, or alogrithms; it is about understanding. -William Paul Thuston
        </p>
      </div>
      <footer
        class="copyrightText"
      >
        <p
          class="copyright-text"
        >
          Copyright © Math Magicians 2023. All rights reserved.
        </p>
      </footer>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
